### PR TITLE
Feature/get remote serverinfo

### PR DESCRIPTION
**Pull request by RuneLind**
ssh login to server and display directory recursively

### CODE REVIEW

File: lib/remote_server.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
+require 'net/ssh'
+
+class RemoteServer
+  class FileTree
+    attr_reader :path
```
Comment (digitalex) [inline lib/remote_server.rb:5]: Just minor nitpicking, but you can do

``` ruby
attr_reader :path, :name
```

**Comment by digitalex** [comment]
Looks good!

File: lib/remote_server.rb (hunk 1 above)
Comment (birgerlie) [inline lib/remote_server.rb:5]: attr_accessor :children, :parent ?

Comment (RuneLind) [inline lib/remote_server.rb:5]: yep wil changed this

**Pull request merged**